### ISSUE TITLE
PHP build failure on CircleCI

### ISSUE THREAD

**Issue by ilammy** [cossacklabs/themis#359]
PHP builds have started failing recently ([PHP 5], [PHP 7.0], [PHP 7.1]), all with the same error:

[PHP 5]: https://circleci.com/gh/cossacklabs/themis/1706
[PHP 7.0]: https://circleci.com/gh/cossacklabs/themis/1705
[PHP 7.1]: https://circleci.com/gh/cossacklabs/themis/1704

```
------------------------------------------------------------
Running phpthemis tests.
If any error, check https://github.com/cossacklabs/themis/wiki/PHP-Howto
------------------------------------------------------------
build/tests/phpthemis_test.sh
ERROR: Invalid installer signature, probably installer was updated
If installation fails, means installer signature is invalid
```

There were no (obvious) changes to the PHP build infrastructure from our side 🤔 The master build was fine [yesterday].

[yesterday]: https://circleci.com/gh/cossacklabs/themis/1696

**Comment by vixentael** [comment]
Should be fixed by #360

**Comment by vixentael** [comment]
I've also updated logs to make it more clear _where_ we should update installer's signature

**Comment by vixentael** [comment]
Closed by #360
